docs/topics/http/sessions.txt

```diff
@@ -72,28 +72,37 @@ If you have multiple caches defined in :setting:`CACHES`, Django will use the
 default cache. To use another cache, set :setting:`SESSION_CACHE_ALIAS` to the
 name of that cache.
 
-Once your cache is configured, you've got two choices for how to store data in
-the cache:
-
-* Set :setting:`SESSION_ENGINE` to
-  ``"django.contrib.sessions.backends.cache"`` for a simple caching session
-  store. Session data will be stored directly in your cache. However, session
-  data may not be persistent: cached data can be evicted if the cache fills
-  up or if the cache server is restarted.
-
-* For persistent, cached data, set :setting:`SESSION_ENGINE` to
-  ``"django.contrib.sessions.backends.cached_db"``. This uses a
-  write-through cache -- every write to the cache will also be written to
-  the database. Session reads only use the database if the data is not
-  already in the cache.
-
-Both session stores are quite fast, but the simple cache is faster because it
-disregards persistence. In most cases, the ``cached_db`` backend will be fast
-enough, but if you need that last bit of performance, and are willing to let
-session data be expunged from time to time, the ``cache`` backend is for you.
-
-If you use the ``cached_db`` session backend, you also need to follow the
-configuration instructions for the `using database-backed sessions`_.
+Once your cache is configured, you have to choose between a database-backed
+cache or a non-persistent cache.
+
+The database cache ``django.contrib.sessions.backends.cached_db`` uses a
+write-through cache -- every write to the cache will also be written to the
+database. Session reads only use the database if the data is not already in the
+cache.
+
+For persistent, cached data, set :setting:`SESSION_ENGINE` to
+``"django.contrib.sessions.backends.cached_db"``. And follow the configuration
+instructions for the `using database-backed sessions`_.
+
+To use the non-persistent cache ``django.contrib.sessions.backends.cache`` set
+:setting:`SESSION_ENGINE` to ``"django.contrib.sessions.backends.cache"`` for a
+simple caching session store. Session data will be stored directly in your
```

Comment:
In a way, "cached_db" also stores "directly", I think "only" is a better word here

```suggestion
simple caching session store. Session data will be stored only in your
```

Reply:
Updated, thanks.